lib/formats/json/json.flow

```diff
@@ -200,52 +199,61 @@ getJsonIntValueM(json : Json) -> Maybe<int> {
 	}
 }
 
-doParseJson(json : string, jsonLength : int, start : int) -> Pair<Json, int> {
+doParseJson(json : string, jsonLength : int, start : int) -> Triple<Json, int, int> {
 	charCode = getCharCodeAt(json, start);
-	if (charCode == 32 || charCode == 9 || charCode == 10 || charCode == 13) { // " " || "\t" || "\n" || "\r"
+	v = if (charCode == 32 || charCode == 9 || charCode == 10 || charCode == 13) { // " " || "\t" || "\n" || "\r"
 		// Eat white space
 		doParseJson(json, jsonLength, start + 1);
 	} else {
 		if (charCode == 91) {				// "["
 			v = parseJsonArray2(json, jsonLength, start + 1, makeList());
-			Pair(JsonArray(v.first), v.second);
+			Triple(JsonArray(v.first), v.second, v.third);
 		} else if (charCode == 123) {		// "{"
 			v = parseJsonObject2(json, jsonLength, start + 1, makeList());
-			Pair(JsonObject(v.first), v.second);
+			Triple(JsonObject(v.first), v.second, v.third);
 		} else if (charCode == 34) {		// "\""
 			v = deserializeRestOfString2(json, start + 1);
-			Pair(JsonString(v.first), v.second);
+			Triple(JsonString(v.first), start+1, v.second);
```

Comment:
Do we really need Triple?
If yes, Triple is bad choice here, the meaning of second and third is not clear. Seems special type is needed like ParseAcc

make clear comment in the code how and why it is used

Reply:
I will rework this, thanks for your advice.

Reply:
I will rework this, thanks for your advice.